contracts/main/partnersAgreement/contracts/PartnersAgreement.sol

```diff
@@ -88,8 +94,17 @@ contract PartnersAgreement is IPartnersAgreement, ERC721Holder {
         coreTeamMembersCount = pa.coreTeamMembersCount;
         interactionsQueryServer = pa.interactionsQueryServer;
 
-        for (uint256 i = 0; i < pa.partnersContracts.length; i++)
-            partnersContracts.push(pa.partnersContracts[i]);
+        for (uint256 i = 0; i < pa.partnersContracts.length; i++) {
+            if(pa.partnersContracts[i] != address(0)) {
+                if(pa.partnersContracts[i] != pa.communityAddress) {
+                    require(
+                            IOwnable(pa.partnersContracts[i]).owner() == pa.owner,
+                            "Only the owner of the contract can import it!"
```

Comment:
all core team members can import a contract

Reply:
I copied that condition from `addNewContractAddressToAgreement()`

Reply:
it's not about who can import but who's owner of the contract. Maybe error message is not correct but again I just copied it

Reply:
Importing partners contracts was implemented before core team members. All core team members can import contracts that they have deployed. Imagine the case when Alex is signing the PA. He can't import contracts because he is not the one deploying them. He would add me as a core team member and I would be the one importing them. The logic in both places should be changed. (When creating PA and when importing a contract)